docs/logs/sending-docker-logs.md

```diff
@@ -0,0 +1,49 @@
+title: Sending Docker Logs with Logagent
```

Comment:
I'm confused.  I see sending-container-logs.md.  Is this file a dupe?

Reply:
I don't know why the other file went to GH, i deleted it with git rm. Sorry.